networking_aci/plugins/ml2/drivers/mech_aci/rpc_api.py

```diff
@@ -59,6 +59,9 @@ def clean_baremetal_objects(self, rpc_context, resource_name):
     def sync_direct_mode_config(self, rpc_context, host_config):
         self.sync_direct_mode_config_agent(host_config)
 
+    def sync_network(self, rpc_context, network):
+        self.sync_network(network)
```

Comment:
How does this not just call itself thus raising a `TypeError` because it needs 3 arguments?